google-cloud-logging/lib/google/cloud/logging/project.rb

```diff
@@ -347,22 +340,15 @@ def async_writer max_queue_size: AsyncWriter::DEFAULT_MAX_QUEUE_SIZE
         #   gcloud = Google::Cloud.new
         #   logging = gcloud.logging
         #
-        #   resource = logging.resource "gae_app", labels: {
-        #                                 "module_id" => "1",
-        #                                 "version_id" => "20150925t173233" }
-        #                               }
+        #   resource = logging.resource "gae_app",
+        #                               module_id: "1",
+        #                               version_id: "20150925t173233"
         #
-        #   logger = logging.logger "my_app_log", resource,
-        #                           labels: {env: :production}
+        #   logger = logging.logger "my_app_log", resource, env: :production
         #   logger.info "Job started."
         #
-        def logger log_name, resource, labels: {}, async_writer: true
```

Comment:
I think it's important to allow the caller to provide an AsyncWriter. A common use case will be to create one AsyncWriter that is shared among N Loggers.